Week2/gradecalc.js

```diff
@@ -0,0 +1,32 @@
+"use strict"
+
+/* the function below is to turn the score into percentual*/
+function perc (score, total) {
+    return score * total / total
+}
+
+/*the function below will put the score into a class of grade*/
+function whichgrade (score) {
+    if (score >= 0 && score <= 49){
+        grade= "F"
+    }
+    if (score >= 50 && score < 59){
+        grade= "E"
+    }
+    if (score >= 60 && score <69){
+        grade= "D"
+    }
+    if (score >= 70 && score <79){
+        grade= "C"
+    }
+    if (score >= 80 && score <89){
+        grade= "B"
+    }
+    else (score >= 90 && score <= 100){
+        grade= "A"
+    }
+
+    return grade
+}
```

Comment:
Ah. Couple things here:
1. You missed declaring `grade` using a keyword such as `let`. 
2. You need not have the first part of the `else-if` conditions as the second part of the && operator should suffice. I would also recommend adding a check at the top to verify if score is a number and is not less than 0 or greater than 100.